ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1689,3 +1689,10 @@ FCU_DSP_DFVSC_DTM_RTM:
   - DTM
   - RTM
 
+FCU_DFMSC_ZTC_RMM_FDPSM:
```

Comment:
`DFMSC` should be accompanied by `DFSS`. I.e. `FCU_DFSS_DFMSC_ZTC_RMM_FDPSM`

Reply:
Could you explain to me why this is the case? i believe all the fields are covered under DFMSC

Reply:
This is the standard for existing canonical types with multi-speed control, however I agree that it is repetitive. We can keep it like this.